### DISCUSSION TITLE
Vercel NextJS Application not loading properly on Mobile but works fine on Desktop, just me?

### DISCUSSION THREAD

**Discussion by mrgreen33gamer** [vercel/next.js#45421] [Help]
Plain and simple, when I load the application on my PC, everything functions properly. However, when I load the application using my iPhone, there are no data pulls, animations don't work, it's almost like everything is frozen except for a loading spinner component I have added. Is anyone else encountering this issue? I have an iPhone 11, iOS version: 14.0

Before Vercel's updates, the application would load just fine on my phone, now it just breaks.

**Comment by icyJoseph** [comment]
Got a link to the application or repository?

**Reply by mrgreen33gamer** [reply]
Yes sir, WacoExchange.com

**Reply by icyJoseph** [reply]
Well that page is responding with 502, tough luck.

**Reply by mrgreen33gamer** [reply]
> Well that page is responding with 502, tough luck.

Sorry about that, I was finishing up the Ubuntu Server last night. The main link should work again, if not, here's the Vercel Application:

https://wacoexchange.vercel.app/

**Reply by texaspacm** [reply]
@mrgreen33gamer did you ever get it working?

**Comment by tuliooov** [comment]
Did you fix? It's doing on my application too

**Reply by Leekkelly** [reply]
did you fix the issues? I have the same problem too

**Reply by jaluprayoga** [reply]
did you fix it? i get the problem too

**Comment by CodyBontecou** [comment]
I was running into this issue. It ended up being a a node issue. Updating the Vercel environment to node 20 and redeploying solved it.

**Comment by erikfiala** [comment]
Same issue here

**Comment by KhadijaMahanga** [comment]
I am having same issue! My vercel environment's node version is already at 20.

**Reply by Leekkelly** [reply]
got a link to the repo?

**Reply by KhadijaMahanga** [reply]
@Leekkelly I found what the problem was. Vercticel authenitcation was enabled under Deployment Protection. I hadn't noticed on my desktop because I had log in on my vercel account

**Comment by parthhsaxena** [comment]
Did anyone find a solution to this?

**Comment by CH-Umar-Aslam** [comment]
The issue with my Next.js application being frozen and unresponsive on mobile devices was resolved by removing a specific font family from my configuration, which was as follows:

`noto: ["'Noto Nastaliq Urdu'", "sans-serif"],`


After removing this font family, the application started working correctly on mobile devices. If you're encountering a similar issue, consider reviewing and adjusting your font configurations.
